qt/widgets/common/inc/MantidQtWidgets/Common/MantidHelpWindow.h

```diff
@@ -24,10 +25,9 @@ class EXPORT_OPT_MANTIDQT_COMMON MantidHelpWindow : public API::MantidHelpInterf
   Q_OBJECT
 
 public:
-  static bool helpWindowExists() { return g_helpWindow != nullptr; }
+  static bool helpWindowExists() { return !g_helpWindow.isNull(); }
 
-  MantidHelpWindow(QWidget *parent = nullptr, const Qt::WindowFlags &flags = nullptr);
```

Comment:
Quick question: do we not want the option of having a parent?

Reply:
The ability to pass a parent widget has been removed for a couple of reasons:

- It is unused elsewhere in the codebase (including for the LoadDialog and AlgorithmDialog which just passes a nullptr)
- It causes a crash (the one in the issue), and a fix would require implementing a better design for the help documentation code, which would be timely. Am I right in thinking the current documentation mechanism in Mantid will be overhauled in the near future?

Reply:
Ah I see. I hadn't realised the parent option wasn't used at all. It's fine to leave it out now. I do want to see if we can remove this heavy weight help engine stuff at some point.